webrtc.html

```diff
@@ -2389,9 +2389,9 @@
           takes a dictionary argument, <var>candidateInitDict</var>, whose content is used
           to initialize the new
           <a class="idlType" href="#idl-def-RTCIceCandidate"><code>RTCIceCandidate</code></a>
-          object. If either <var>sdpMid</var> or <var>sdpMLineIndex</var> is not present in
+          object. If the <var>sdpMLineIndex</var> is not present in
           <var>candidateInitDict</var>, the corresponding attribute will be initialized to
-          <code>null</code>. If neither is present, a <code>TypeError</code> exception
+          <code>null</code>. If <var>sdpMid</var> is not present, a <code>TypeError</code> exception
           will be thrown.</dd>
```

Comment:
Throwing a TypeError if a member is not present, as the prose says, is the same behaviour as requiring it. We can also have a default value on the sdpMLineIndex member to have it initialized to null if not present.
```
required DOMString sdpMid;
unsigned short? sdpMLineIndex = null;
```